### DISCUSSION TITLE
Who is the Teddy 🐻 named after?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#983] [Galaxy Trivia] [answered]
**Bonus:** Where is the world's largest Teddy 🐻?

**Comment by Sriansh-raj** [comment, marked as answer]
The Teddy Bear is named after former United States President **Theodore Roosevelt**
. 
In 1902, Roosevelt participated in a bear-hunting trip in Mississippi. During the trip, Roosevelt refused to kill a bear that had been captured by his assistants, viewing it as unsportsmanlike. This event was widely reported in newspapers across the country, and political cartoonist Clifford Berryman decided to satirize the president's refusal to shoot the bear. Morris Michtom, a Brooklyn candy shop owner who made stuffed animals, saw the cartoon and created a stuffed toy bear dedicated to the president who refused to shoot a bear. He called it "Teddy's Bear"
. After receiving Roosevelt's permission to use his name, Michtom mass-produced the stuffed bears, which became an instant success
.

The world's largest teddy bear was made in **Mexico** in April 2019 and measures 19.41 meters (63 feet 8 inches) in length.

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/Sriansh-raj?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Reply by Sriansh-raj** [reply]
Thank You sooooo much @ParanoidUser ❤️
